### ISSUE TITLE
Numbered list/head in Pod (S26)

### ISSUE THREAD

**Issue by tushardave26** [perl6/specs#117]
Hello,

I was referring to S26 to get familiarity with Perl 6 Pod system. I am trying the examples from the synopsis. Unfortunately, the numbered list/head code doesn't work the way the doc said. I have tried all the combination that explained in the doc. Below is what I have tried so far.

**Try -- 1**
```
=begin pod
=for head1 :numbered
The Problem

=for head1 :numbered
The Solution

=end pod
```

**Try -- 2**
```
=begin pod
=config head1 :numbered
=config head2 :numbered

=head1 The Problem
=head1 The Solution

=end pod
```

**Try -- 3**
```
=begin pod

=head1 # The Problem
=head1 # The Solution

=end pod
```

None of the above return the heading with numbers. I have also tried with all of them with item instead head. With item, they return all item with * bullet style and not the number.

Am I doing something wrong? or something changed. Can someone please help me?

Thanks.

**Comment by tbrowder** [comment]
On Saturday, September 3, 2016, Tushar Harishbhai Dave <
notifications@github.com> wrote:

> Hello,
>
> I was referring to S26 to get familiarity with Perl 6 Pod system. I am
> trying the examples from the synopsis. Unfortunately, the numbered
> list/head code doesn't work the way the doc said. I have tried all the
> combination that explained in the doc. Below is what I have tried so far.
>
Sorry to say that not all features of Perl 6 pod are implemented yet.  To
get a feel for what is implemented, look at the pod6 files that produce
docs.perl6.org.

Best regards,

-Tom

>

**Comment by tushardave26** [comment]
Thanks Tom for your prompt response. Could you please direct me the location of the Pod files that you mentioned?

And it's very unfortunate not to have those features. Looking forward to seeing them implemented soon. 

-Tushar

**Comment by tbrowder** [comment]
On Sunday, September 4, 2016, Tushar Harishbhai Dave <
notifications@github.com> wrote:

> Thanks Tom for your prompt response. Could you please direct me the
> location of the Pod files that you mentioned?
>
...

You're welcome, Tushar.  For the pod source, check out the git repo at <
https://github.com/perl6/doc>.

A good start there is to look at the pod6 files in the doc/Language
subdirectory.

BTW, I assume you have been checking out the official docs at docs.perl6.org
.

> And it's very unfortunate not to have those features. Looking forward to
> seeing them implemented soon.

Well, I'm working on getting tables fully functional, but I haven't worked
on the project for a while.  When I finish that successfully, I plan to
work on getting other portions of inoperative pod to work.  You can find my
work in my fork of rakudo at <https://github.com/tbrowder/rakudo>.

Best regards,

-Tom

**Comment by tushardave26** [comment]
Thanks again Tom for your reply.

Yes, docs.perl6.org is the first place where I usually look for the stuff.

If you can explain to me how to start contributing to the project, I am more than happy to help. I have never contributed to any open source project before so I am in kind of fuzzy state. I can contribute in my spare time. 

-Tushar

**Comment by tbrowder** [comment]
On Sunday, September 4, 2016, Tushar Harishbhai Dave <
notifications@github.com> wrote:

> Thanks again Tom for your reply.
>
> Yes, docs.perl6.org is the first place where I usually look for the stuff.
>
> If you can explain to me how to start contributing to the project, I am
> more than happy to help. I have never contributed to any open source
> project before so I am in kind of fuzzy state. I can contribute in my spare
> time.
>
Well, Tushar, depending on your experience, it may take a bit of effort to
learn the ropes, but there are all kinds of things that one can do to
help.  And the community is basically nice and helpful.

(I assume you have already installed Perl 6 and are experimenting with it.)

Several places to start, IMHO (in no particular order):

1.  get a github.com account and get familiar with git

2.  join the #perl6 IRC channel and start following conversations, jump in
and introduce yourself when you feel comfortable

3.  check out the "how to help" page on perl6.org:

  http://perl6.org/getting-started/involved

4.  subscribe to all the mailing lists

Welcome to the Perl 6 community, Tushar!

Best regards,

-Tom

**Comment by tushardave26** [comment]
Thanks Tom for your informative comments. 

To just give you information about my background, I am an active Perl 5 user and exploring Perl 6 since its first launch. I use Perl5 in my everyday job as it's the first language that I have learned and worked with. I already have a GitHub account and using it for my projects. I have already signed up for Perl 6 weekly and also asked questions over IRC channel. I believe, now I have adequate knowledge of Perl 6 to start up something on my own. So, I have started developing Perl 6 module. I hope I will get some time to finish it soon. I am hoping to see Perl 6 book out soon as I have faced a lot of trouble finding a step by step guide for myself for Perl 6. I have attended YAPC::NA 2016 where Larry said he has no plan to write a book in near future. But the book will be a great thing at this moment (as per my point of view).

Thanks again for all your replies.

**Comment by tbrowder** [comment]
On Sunday, September 4, 2016, Tushar Harishbhai Dave <
notifications@github.com
<javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:

> Thanks Tom for your informative comments.
>
You're welcome, Tushar, glad they're useful.

> To just give you information about my background, I am an active Perl 5
> user and exploring Perl 6 since its first launch.
>
...

 I'm in similar situation: using Perl 4, then 5, for work and fun since
1993. Hoping for Perl 6 sooner, but using it since early 2015--and not
looking back (except when necessary).

I'm actively working on two Perl 6 modules and have another on deck.  Also
contributing to p6 docs, and rakudo code where I can.

> I am hoping to see Perl 6 book out soon as I have faced a lot of trouble
> finding a step by step guide for myself for Perl 6. I have attended
> YAPC::NA 2016 where Larry said he has no plan to write a book in near
> future. But the book will be a great thing at this moment (as per my point
> of view).
>
I, too, look forward to a book.  Tushar, I was at YAPC::NA 2016, too!  Did
you attend any of Damian's classes?  Either in a hallway meeting with Larry
Wall and Damian, or from Damian, or through the  grapevine (I think Damian
said it in class because I asked about a book), Damian and Larry are
talking with O'Reilly about a book to come out early next year.

What name are you using on #perl6?  I'm "tbrowder" (I gave up on silly
names decades ago).  What part of the world do you live in?

I'm trying to get interest in Perl ( esp. Perl 6) in northwest Florida.  We
do have a Linux User Group here (https://nwflug.org), and a Python
group, but not much Perl interest.

> Thanks again for all your replies.
>
My pleasure!  Happy Perl Sixing!

-Tom

>

**Comment by tushardave26** [comment]
No, I haven't attended any of Damian's classes because of either the time or the cost. I have attended his talk, though. I am a big fan of his work.

> Damian and Larry are talking with O'Reilly about a book to come out early next year.

This is great news. In the Q&A session with Larry, Larry said he doesn't find enough time to write the book as all of us ask him to attend various conferences.

> What name are you using on #perl6?  I'm "tbrowder" (I gave up on silly
names decades ago).  What part of the world do you live in?

I am not that much active on IRC. Whenever I am on it, I use "tushar". I am in Baltimore, MD.

> I'm trying to get interest in Perl ( esp. Perl 6) in northwest Florida.  We
do have a Linux User Group here (https://nwflug.org), and a Python
group, but not much Perl interest.

Same here in Baltimore. We have only 5-7 people in MeetUp group. And I think it's same everywhere. Whenever I said to others that I am Perl programmer, first of all, they stare at  me and then ask me for good reasons of why am I still using it. Even few of them ask me to change my interest to Python saying that Perl is not OOP language.

Anyways, nice talking to you.
